packages/jest-runtime/src/index.ts

```diff
@@ -1363,18 +1366,12 @@ class Runtime {
       });
     })();
 
+    const path = this._path;
+
     Object.defineProperty(moduleRequire, 'main', {
       enumerable: true,
       get() {
-        let mainModule = from.parent;
-        while (
-          mainModule &&
-          mainModule.parent &&
-          mainModule.id !== mainModule.parent.id
-        ) {
-          mainModule = mainModule.parent;
-        }
-        return mainModule;
+        return path;
       },
```

Comment:
instead of a getter, just `value: this._path` seems cleaner?

Reply:
Definitely.